### DISCUSSION TITLE
not able to push app to heroku

### DISCUSSION THREAD

**Discussion by sureshchidella** [anasty17/mirror-leech-telegram-bot#1164] [Q&A]
![Screenshot (41)](https://user-images.githubusercontent.com/92628301/197442910-d1ed6cef-8d3e-492a-bde6-36d9431fcbd2.png)
![Screenshot (42)](https://user-images.githubusercontent.com/92628301/197442918-45a490db-79c6-4b06-a090-6c40f834153b.png)
anas bro please check these screenshots of github and heroku logs 
please response to me

**Comment by anasty17** [comment]
No heroku support

**Reply by sureshchidella** [reply]
how can i fix that
btw thanks for your reply

**Reply by anasty17** [reply]
It's not an issue to fix kid!
